husky_description/urdf/description.xacro

```diff
@@ -31,7 +31,7 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
   <xacro:arg name="kinect_enabled" default="false" />
   <xacro:arg name="robotiq_grippers_enabled" default="$(optenv ROBOTIQ_GRIPPERS_ENABLED false)" />
  
-  <xacro:include filename="$(find husky_description)/urdf/husky.urdf.xacro" />
+  <xacro:include filename="$(find husky_description)/urdf/dual_arm_husky.urdf.xacro" />
```

Comment:
Hi @TheDash - this change just killed our Husky, cf. support request #4137 (we spoke to Martin). I've manually made this change to the provided husky installation, rebuilt and it solved our problem (it lost comms due to a malformed URDF - base_laser was not unique causing the husky_node to crash)